server.js

```diff
@@ -108,6 +108,16 @@ utils.addCheckedFunction(nunjucksAppEnv)
 // Redirect root to /docs
```

Comment:
this comment is for the middleware below (line 121) maybe move and have a new one for this middleware?

Reply:
👍 fixed